grades.html

```diff
@@ -9,7 +9,7 @@
 <body>
     <div class="container">
         <h1>Percentage to Grade Converter</h1>
-        <input type="number" id="percentage" placeholder="Enter percentage" min="0" max="100">
+        <input type="text" id="percentage" placeholder="Enter percentage">

```

Comment:
Good point if they're getting extra credit, but they would be surpassing an A+ grade by common knowledge.

Also, why are we changing the input type to text if were entering numbers?